package.json

```diff
@@ -23,29 +23,29 @@
     "all": "npm run format && npm run lint && npm run package && npm test"
   },
   "dependencies": {
-    "@actions/core": "^1.2.6",
+    "@actions/core": "^1.10.0",
```

Comment:
Verified this is what github recommends to get the latest get-output: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#patching-your-actions-and-workflows